jbi/models.py

```diff
@@ -69,7 +71,7 @@ class ActionParams(BaseModel):
     jira_project_key: str
     steps: ActionSteps = ActionSteps()
     jira_components: JiraComponents = JiraComponents()
-    labels_brackets: str = Field("no", enum=["yes", "no", "both"])
+    labels_brackets: Literal["yes", "no", "both"] = "no"
```

Comment:
`Field` with `enum` isn't supported anymore, but I think the change expresses the same thing.